### PR TITLE
Add support for Leap 15.3 ARM (which is armv7 only)

### PR DESCRIPTION
**Pull request by ggardet**
and backup 15.2 ARM files

Note: Not sure why `make test_regen_all` clears all *.lst files...

### CODE REVIEW

**Review by andrii-suse — APPROVED** [review]

File: xml/obs/openSUSE:Leap:15:ARM.xml [hunk 1]
```diff
@@ -1,11 +1,11 @@
...
-    <flavor name="NET|DVD" folder="*product*" distri="opensuse" iso="1">
+    archs="armv7hl">
+    <flavor name="NET" folder="*product*" distri="opensuse" iso="1">
         <repos>
-            <oss folder="openSUSE-ftp-ftp" debug="{java*,kernel-default-debug*,kernel-default-base-debug*,mraa-debug*}" source="{coreutils*,yast2-network*}" mirror="1" dest="openSUSE-15.2-oss-aarch64"/>
+            <oss folder="Leap-ftp-ftp" debug="{java*,kernel-default-debug*,kernel-default-base-debug*,mraa-debug*}" source="{coreutils*,yast2-network*}" mirror="1" dest="openSUSE-Leap-${version}-oss-$arch"/>
         </repos>
     </flavor>
-    <news iso="DVD" archs="aarch64"/>
+<!--     <news iso="DVD" archs="armv7hl"/> -->
```
Comment (Martchus) [inline xml/obs/openSUSE:Leap:15:ARM.xml:10]: Leftover?

Comment (ggardet) [reply xml/obs/openSUSE:Leap:15:ARM.xml:10]: I left it to think about it on next update.

**Comment by Vogtinator** [comment]
This includes `openSUSE:Leap:15.2:ARM(:Images)`?

**Comment by ggardet** [comment]
> This includes `openSUSE:Leap:15.2:ARM(:Images)`?

`openSUSE:Leap:15.2:ARM(:Images)` are in separate files, part of this PR. Because Leap 15.3 is quite different from Leap 15.2 regarding architectures.

**Comment by Vogtinator** [comment]
> > This includes `openSUSE:Leap:15.2:ARM(:Images)`?
> 
> `openSUSE:Leap:15.2:ARM(:Images)` are in separate files, part of this PR. Because Leap 15.3 is quite different from Leap 15.2 regarding architectures.

Right. I was just a bit confused because the description said `and backup 15.2 ARM files`, which I couldn't quite match up with the diff.

**Comment by andrii-suse** [comment]
This is waiting for obs to be ready, because .iso is not there yet

**Pull request merged**